DeviceAdministration/Infrastructure.UnitTests/Infrastructure/DeviceLogicTests.cs

```diff
@@ -47,14 +46,14 @@ public DeviceLogicTests()
         [Fact]
         public async void GetDevicesTest()
         {
-            DeviceListQuery q = fixture.Create<DeviceListQuery>();
-            DeviceListQueryResult r = fixture.Create<DeviceListQueryResult>();
+            var q = fixture.Create<DeviceListQuery>();
```

Comment:
Please don't use single char variable names. It gets hard to read down the line on what it is supposed to be. Should be descriptive enough for readability.